AGENT-863: docs fix (#8325)
openshift/installer: docs/user/agent/add-node/add-nodes.md

## Pre-requisites
1. The `oc` tool must be available in the execution environment (the "user host").

> 2. Ensure `oc` is properly configured to manage the cluster to be expanded, ie by either:

Reviewer: ```suggestion
2. Ensure `oc` is properly configured to manage the cluster to be expanded, i.e. by either:
```

>    - running `oc login` command with the required cluster credentials

Reviewer: ```suggestion
   - running `oc login` command with the required cluster credentials, or
```

>    - export the environment variable `$KUBECONFIG` pointing to the cluster `kubeconfig` file

Reviewer: ```suggestion
   - running `export KUBECONFIG=<cluster `kubeconfig` file>`
```
